policy_definitions/tagging/policy_set_definition_qby_require_tags.tmpl.json

```diff
@@ -0,0 +1,39 @@
+{
+    "name": "QBY-Define-Require-Tags",
```

Comment:
```suggestion
    "name": "QBY-Require-Tags",
```
there is no harm in the same name of policy and policy assignment

Reply:
Changed as suggested.